protocol/messenger.go

```diff
@@ -5724,3 +5725,26 @@ func (m *Messenger) handleSyncSocialLinks(message *protobuf.SyncSocialLinks, cal
 func (m *Messenger) GetDeleteForMeMessages() ([]*protobuf.SyncDeleteForMeMessage, error) {
 	return m.persistence.GetDeleteForMeMessages()
 }
+
+func (m *Messenger) startMessageSegmentsCleanupLoop() {
+	logger := m.logger.Named("messageSegmentsCleanupLoop")
+
+	go func() {
+		var interval time.Duration = 0
+		for {
+			select {
+			case <-time.After(interval):
+				// Immediate execution on first run, then set to regular interval
+				interval = 1 * time.Hour
```

Comment:
Do we really need a loop? Just doing it once per launch should be enough? I guess in case someone never closes the app it's good :laughing:

Reply:
I think that's both ok, the one thing we might not want to do, is to schedule it right at launch time, I would give it say a minute or two, since at start up there's a lot happening, the drawback is that if someone just opens it for 59 seconds it might not run, but I think that's acceptable.

Reply:
Valid point. Addressed: https://github.com/status-im/status-go/pull/4328